### PR TITLE
PHP Timeout

### PR DESCRIPTION
**Pull request by mckernanin**
Added a setting to specify the PHP timeout. I'm testing this on my own machine, wanted to share to see if this will fix our "PHPCS ate my battery" issues.

### CODE REVIEW

**Comment by Arcanemagus** [comment]
What's the purpose of this? There is already a "Disable timeout" setting for those that need to run past 10 seconds, and for most people it should be catching this already with the 10 second timeout... which apparently isn't working on some systems.

This PR will just allow you to change when it doesn't work, it won't fix the underlying problem which is either in [`sb-exec](https://github.com/steelbrain/exec) or Node.js itself.

File: lib/main.js [hunk 1]
```diff
@@ -278,6 +284,8 @@ export default {
         };
         if (disableExecuteTimeout) {
           execOptions.timeout = Infinity;
+        } else if (executeTimeout) {
+          execOptions.timeout = executeTimeout;
```
Comment (Arcanemagus) [inline lib/main.js]: Just set this up in the initial execOptions declaration, it will _always_ have a value due to this being observed.

**Comment by Arcanemagus** [comment]
Thanks for taking the time to submit a PR that would have worked btw!

**Comment by mckernanin** [comment]
Glad we have a resolution for this! The batteries of MacBooks everywhere thank you 👍

**Pull request closed without merging**